File: src/Entities/Medicine.Entities/Models/Reminder.cs
using Medicine.Entities.Enums;
using Medicine.Entities.Models.Base;

namespace Medicine.Entities.Models
{
    /// <summary>
    /// User create reminder select time and then can assing drugs for this reminder 
    /// for example each day in 17:00
    /// </summary>
    public class Reminder : EntityTitleDescription
    {
        public string TimeInUtc { get; set; } = "00:00";
        public List<DosingFrequencyReminder> DosingFrequencyReminders { get; set; } = new List<DosingFrequencyReminder>(); 

    }
}

File: front/medicine/src/app/shared/models/entity/Reminder.ts
import { User } from "./User";
import { DosingFrequencyReminder } from "./DosingFrequencyReminder";

export class Reminder {
  timeInUtc!: string;
  dosingFrequencyReminders: DosingFrequencyReminder[] = [];
  title?: string;
  description?: string;
  user?: User;
  userId?: number;
  id!: number;
  createdAt?: Date;
}
